libraries/transformer/src/androidTest/java/androidx/media3/transformer/mh/HdrEditingTest.java

```diff
@@ -139,6 +139,41 @@ public void export_transmuxHlg10File() throws Exception {
     assertThat(actualColorTransfer).isEqualTo(C.COLOR_TRANSFER_HLG);
   }
 
+  @Test
+  public void export_transmuxDolbyVisionFile() throws Exception {
+    Context context = ApplicationProvider.getApplicationContext();
+
+    if (Util.SDK_INT < 24) {
+      // TODO: b/285543404 - Remove suppression once we can transmux H.265/HEVC before API 24.
+      recordTestSkipped(context, testId, /* reason= */ "Can't transmux H.265/HEVC before API 24");
+      return;
+    }
+
+    if (AndroidTestUtil.skipAndLogIfFormatsUnsupported(
+        context,
+        testId,
+        /* inputFormat= */ MP4_ASSET_DOLBY_VISION_HDR_FORMAT,
+        /* outputFormat= */ null)) {
+      return;
+    }
+
+    Transformer transformer = new Transformer.Builder(context).build();
+    MediaItem mediaItem = MediaItem.fromUri(Uri.parse(MP4_ASSET_DOLBY_VISION_HDR));
+
+    ExportTestResult exportTestResult =
+        new TransformerAndroidTestRunner.Builder(context, transformer)
+            .build()
+            .run(testId, mediaItem);
+    assert exportTestResult.filePath != null;
```

Comment:
We generally use `assertThat(xyz).isNotNull();`

Reply:
OK, I'll change it.

Reply:
Based on your latest comment, I'll remove this sentence.